src/content/docs/fr/guides/cms/flotiq.mdx

```diff
@@ -0,0 +1,294 @@
+---
+title: Flotiq & Astro
+description: Ajouter du contenu à votre projet Astro en utilisant Flotiq comme CMS
+type: cms
+service: Flotiq
+i18nReady: true
+---
+
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro';
+import { Steps } from '@astrojs/starlight/components';
+
+[Flotiq](https://flotiq.com) est un CMS headless conçu pour différents frontends, tels que les sites statiques, les applications mobiles et les applications web. Les développeurs et les créateurs de contenu gèrent et diffusent le contenu par le biais d'API REST et GraphQL.
+
+## Intégration avec Astro
+
+Ce guide permet d'utiliser l'API Flotiq headless CMS avec un projet Astro pour afficher du contenu sur votre site web.
+
+### Pré-requis
+
+Pour commencer, vous aurez besoin de
+
+1. **Un projet Astro** - Vous pouvez créer un nouveau projet en utilisant la commande `npm create astro@latest`.
+2. **Un compte Flotiq** - Si vous n'avez pas de compte, [inscrivez-vous gratuitement](https://editor.flotiq.com/register).
+3. **Une clé API en lecture seule Flotiq** - Découvrez [comment obtenir votre clé](https://flotiq.com/docs/API/).
+
+### Configuration des variables d'environnement
+
+Ajoutez la clé API en lecture seule de votre compte Flotiq au fichier `.env` à la racine de votre projet Astro :
+
+```ini title=".env"
+FLOTIQ_API_KEY=__YOUR_FLOTIQ_API_KEY__
+```
+
+### Définir un type de contenu dans Flotiq
+
+Tout d'abord, vous devez définir un exemple de [Content Type Definition](https://flotiq.com/docs/panel/content-types/) dans Flotiq pour stocker des données.
+
+Connectez-vous à votre compte Flotiq et créez une définition de type de contenu personnalisée avec l'exemple de configuration `Blog Post` suivant :
+- **Label** : Blog Post
+- **API Name** : blogpost
+- **Fields** :
+  - **Title** : texte, obligatoire
+  - **Slug** : texte, obligatoire
+  - **Content** : texte enrichi, obligatoire
+
+Ensuite, créez un ou plusieurs exemples d'[objets de contenu](https://flotiq.com/docs/panel/content-objects/) en utilisant le type « Blog Post ».
```

Comment:
nit, the original version uses backticks and in the previous occurrence you left the backticks, so:
```suggestion
Ensuite, créez un ou plusieurs exemples d'[objets de contenu](https://flotiq.com/docs/panel/content-objects/) en utilisant le type `Blog Post`.
```